### PR TITLE
updated changelogs and ver pulled in by template

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

**Review by calebmshafer — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@ jobs:
           echo 'Installing...'
           node common/scripts/install-run-rush.js install
 
-          # echo 'Checking Rush Change...'
-          # node common/scripts/install-run-rush.js change -v -b origin/master
+          echo 'Checking Rush Change...'
```
[the remark this answers is not in the document]
Comment (kckst8) [reply .github/workflows/ci.yml:29]: disconnected head......you guys used to get around it through a custom script if i recall

[the remark this answers is not in the document]
Comment (kckst8) [reply .github/workflows/ci.yml:29]: image

[the remark this answers is not in the document]
Comment (aruniverse) [reply .github/workflows/ci.yml:29]: think we can just do `change -v`

File: packages/modules/cra-template-desktop-viewer/template.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
       "@itwin/core-quantity": "^3.0.0",
       "@itwin/core-react": "^3.0.0",
       "@itwin/core-telemetry": "^3.0.0",
-      "@itwin/desktop-viewer-react": "next",
+      "@itwin/desktop-viewer-react": "latest",
```
[the remark this answers is not in the document]
Comment (aruniverse) [reply packages/modules/cra-template-desktop-viewer/template.json:23]: In CHANGELOG-2.0.md How to upgrade an existing application, should i just pin the dep down to 2 too?

[the remark this answers is not in the document]
Comment (kckst8) [reply packages/modules/cra-template-desktop-viewer/template.json:23]: i think so

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kckst8) [reply .github/workflows/ci.yml:29]: yeah i battled with this for hours when we first setup the repo.

File: CHANGELOG-2.0.md [hunk 3]
```diff
@@ -72,12 +72,12 @@ We've replaced the default measure-tools provided in the Viewer from itwinjs-cor
...
 - A [codemod](https://github.com/iTwin/codemods) has been developed to help you upgrade your exisitng code from iModel.js@2.x to iTwin.js@3.0.
 
-- Change the version of `@itwin/web-viewer-react` in your package.json to use the `"latest"` dist tag, eg:
+- Change the version of `@itwin/web-viewer-react` in your package.json to the following:
 
   ```
     "dependencies": {
       ...
-      "@itwin/web-viewer-react": "latest",
+      "@itwin/web-viewer-react": "2",
```
Comment (kckst8) [inline CHANGELOG-2.0.md:80]: should this be ^2.0.0? or is this some shorthand that I am not aware of

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (skirby1996) [reply .github/workflows/ci.yml:29]: Your error ended up pointed me towards the issue in the core repo, I ended up following the advice in [this thread](https://stackoverflow.com/questions/64957915/github-actions-git-diff-master-head-fatal-no-merge-base) and tweaked the fetch depth for the checkout step in our pipeline and that seems to have fixed it. You could try setting 'fetchDepth' to 0 for now: https://github.com/actions/checkout#usage

File: CHANGELOG-2.0.md (hunk 3 above)
Comment (aruniverse) [reply CHANGELOG-2.0.md]: im pretty sure this was a way to specify ^2.x, but ill be more explicit

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aruniverse) [reply .github/workflows/ci.yml]: going to disable for now

**Review by kckst8 — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aruniverse) [reply .github/workflows/ci.yml:31]: @skirby1996 thanks, that worked!

**Pull request merged**